docs/platform/webcontainers/browser-support.md

```diff
@@ -12,25 +12,39 @@ head:
 
 _Last update: February 2023_
 
-**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in beta. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
+## Supported browsers
 
-:::warning Note
-There is a reported Chrome regression on Macbooks with M1 chip, which also affects the speed of some larger projects on WebContainers. Learn more about this issue in these bug reports: [issue 1228686](https://bugs.chromium.org/p/chromium/issues/detail?id=1228686) and [issue 1356099](https://bugs.chromium.org/p/chromium/issues/detail?id=1356099).
-:::
+WebContainers are supported in all recent desktop browsers (fully in Chrome and Chrome-based browsers, in beta in Firefox and Safari), and partially supported on Android browsers.
+
+### Desktop browsers
+
+- Chrome: full support.
+- Other Chromium-based browsers: full support. Some browsers may apply restrictive rules by default, and require specific configuration, like [Brave](#brave).
+- Firefox: beta support ([see details](#firefox)).
+- Safari: beta support since Safari 16.4, no support for earlier versions ([see details](#safari)).
+
+### Mobile browsers
+
+- Android: beta support for Chrome, Chromium-based browsers and Firefox. Depending on your device, large projects may run into memory limitations.
+- iOS and iPadOS: not supported yet. WebContainers can theoretically work on iOS 16.4 and later, but memory limitations for web apps in Safari for iOS and iPadOS are currently too strict to run most WebContainers-based projects.
```

Comment:
@d3lm I rewrote the info about iOS. Does that look good, or do we want to rework it a bit?

Reply:
Yep I think that is accurate as of right now, at least until we test and possibly deploy a separate build for iPadOS.